etc/Dockerfile

```diff
@@ -36,9 +36,18 @@ ENV LANG en_US.UTF-8
 COPY --from=builder /usr/src/grin/target/release/grin /usr/local/bin/grin
 
 WORKDIR /root/.grin
-RUN grin server config && \
+
+ARG grinnet=""
+
+RUN grin ${grinnet} server config && \
     sed -i -e 's/run_tui = true/run_tui = false/' grin-server.toml
 
+VOLUME ["/root/.grin"]
```

Comment:
The volume cmd is unnecessary?

Reply:
Nope, it's useful .. When it's not marked as a volume .. docker does not copy data from inside the image to that location .. so the grin-server.toml is **not** copied to that external location. While when marked as a volume, it is. When it's marked as a volume, we should pass a volume name like `docker run ... -v dotgrin:/root/.grin` .. and see the volume with `docker volume ls`

AFAICT, this is how docker should be used! And it works well. When bind mounting a volume -v .:/root/.grin .. docker does NOT copy the grin-server.toml file.